### ISSUE TITLE
Include page or template with the missing attachment warning.

### ISSUE THREAD

**Issue by abea** [apostrophecms/apostrophe#1630]
`Template warning: Impossible to retrieve the attachment url since it is missing, a default icon has been set. Please fix this ASAP!`

This warning would be 1000% more help if it included either the template or (even better) URL where it was being thrown. Last I looked at this the method throwing this is a few levels separated and doesn't receive information about the template.

**Comment by abea** [comment]
We have got to get some kind of additional information in this. The warning can be triggered when the image isn't actually on the page, so it's incredibly hard to find the source.

**Comment by boutell** [comment]
Agreed it would be very helpful to know the template name here. That could be difficult in Nunjucks 2, not sure about 3.

**Comment by Yashmishra06** [comment]
If the issue still persists I can help out in this . Please assign this to me

**Comment by BoDonkey** [comment]
I Think this issue has been solved. Closing.
